### DISCUSSION TITLE
Control map resolution?

### DISCUSSION THREAD

**Discussion by bvz2000** [domlysz/BlenderGIS#591] [Q&A]
I would like to capture a larger area, but still with the highest possible resolution map.

As far as I can tell, I have to zoom in until I see the map fidelity that I want, then I can capture that area. But I would like to capture a larger area, but still maintain this map fidelity. Is there a way to download the highest res maps, but do it across a larger geographic area?

Alternatively, would it be possible to perfectly tile different areas? I tried to do this manually - shifting my viewport from location to location in an attempt to download adjacent, high res tiles.  But I wind up with several overlapping tiles, and they do not line up perfectly with each other (meaning I cannot trim them very easily to be seamlessly matched going from one tile to another).

Thanks! and an amazing tool!

**Comment by basicallyImjustgonnanottakethevaccine** [comment]
Read manual, you press L and scroll with mouse

**Comment by derricksample** [comment]
Pressing L and scrolling with the mouse doesn't seem to work anymore. It will load the underlying images, but you cannot export until you unlock it, and when you unlock it, it jumps to the closer zoom.

**Reply by basicallyImjustgonnanottakethevaccine** [reply]
Usually I scroll with the mouse then press E to export but if it doesn't work like you said, look for the texture file and open the folder, then delete the map, and create a new one. Then you can get the image at full resolution.
Rn I'm using real world terrain in unity since I experience many crash with blender gis and also I had to do a lot of trick to get what I want.
